.github/workflows/cicd.yml

```
@@ -97,10 +97,13 @@ jobs:
             ghcr.io/hertzg/tesseract-server
             hertzg/tesseract-server
           platforms: |
+            linux/386
             linux/amd64
```

`linux/386` is no longer supported on node (and in linux kernel I think)

```suggestion
            linux/amd64
```